tests/RelationsTest.php

```diff
@@ -327,20 +302,27 @@ public function testBelongsToManyAttachesExistingModels(): void
     public function testBelongsToManySync(): void
     {
         // create test instances
-        $user    = User::create(['name' => 'John Doe']);
-        $client1 = Client::create(['name' => 'Pork Pies Ltd.'])->_id;
-        $client2 = Client::create(['name' => 'Buffet Bar Inc.'])->_id;
+        $user    = User::create(['name' => 'Hans Thomas']);
```

Comment:
We removed the annoying rule [MultipleStatementAlignment](https://github.com/mongodb/laravel-mongodb/blob/4.1/phpcs.xml.dist#L38). You can modify this line.
```suggestion
        $user = User::create(['name' => 'Hans Thomas']);
```